provider/balance_checker.go

```diff
@@ -83,6 +102,82 @@ func (bc *balanceChecker) doCheck(ctx context.Context) (bool, error) {
 	return tooLow, nil
 }
 
+func (bc *balanceChecker) runEscrowCheck(ctx context.Context, lid mtypes.LeaseID, res chan<- leaseCheckResponse) {
+	go func() {
+		select {
+		case <-bc.lc.Done():
+		case res <- bc.doEscrowCheck(ctx, lid):
+		}
+	}()
+}
+
+func (bc *balanceChecker) doEscrowCheck(ctx context.Context, lid mtypes.LeaseID) leaseCheckResponse {
+	resp := leaseCheckResponse{
+		lid: lid,
+	}
+
+	var syncInfo *tmrpc.SyncInfo
+	syncInfo, resp.err = bc.session.Client().NodeSyncInfo(ctx)
+	if resp.err != nil {
+		return resp
+	}
+
+	if !syncInfo.CatchingUp {
```

Comment:
This check seems backwards? If the node is catching up (set to true), we would want to skip everything